ansible/tasks/logging/README.md

```diff
@@ -0,0 +1,34 @@
+# Playbooks to install FluentD and FluentD plugins
```

Comment:
There is a lot of overlap with this README and the one introduced in #21. Please work together and ensure that information is not repeated in multiple places. Ideally there should be a main README with common info and links to the individual READMEs for different playbooks.

Reply:
@sebsoto and I have decided to keep one README file for WSU and logging, so I will be removing the README file from this PR and create a separate PR for the same